api/batches/batch_handler.go

```diff
@@ -271,15 +272,19 @@ func (handler *batchHandler) getAllBatches() (models.JobList, error) {
 	return slice.PointersOf(kubeBatches.Items).([]*batchv1.Job), nil
 }
 
-func (handler *batchHandler) buildBatchJobSpec(namespace, appName, batchName string, radixJobComponent *radixv1.RadixDeployJobComponent, batchScheduleDescriptionSecret *corev1.Secret) (*batchv1.Job, error) {
+func (handler *batchHandler) buildBatchJobSpec(namespace, appName, batchName string, radixJobComponent *radixv1.RadixDeployJobComponent, batchScheduleDescriptionSecret *corev1.Secret, batchScheduleDescription *models.BatchScheduleDescription) (*batchv1.Job, error) {
 	container := handler.getContainer(batchName, radixJobComponent, batchScheduleDescriptionSecret,
 		handler.common.SecurityContextBuilder)
 	volumes := getVolumes(batchScheduleDescriptionSecret)
 	podSecurityContext := handler.common.SecurityContextBuilder.BuildPodSecurityContext(radixJobComponent)
 
+	jobCount := len(batchScheduleDescription.JobScheduleDescriptions)
```

Comment:
batchScheduleDescription is nilable. If passed argument is nil, then len() will panic

Reply:
fixed